fe/src/main/java/org/apache/doris/analysis/SelectStmt.java

```diff
@@ -974,16 +1009,24 @@ private void analyzeAggregation(Analyzer analyzer) throws AnalysisException {
             LOG.debug("desctbl: " + analyzer.getDescTbl().debugString());
             LOG.debug("resultexprs: " + Expr.debugString(resultExprs));
         }
+        /*
+         * All of columns of result and having clause are replaced by new slot ref which is bound by top tuple of agg info.
+         * For example:
+         * ResultExprs: SlotRef(k1), FunctionCall(sum(SlotRef(k2)))
```

Comment:
why do you add FunctionCall to decorate sum(SlotRef(k2))？And There is no function at below “After rewritten” 。

Reply:
Em...The "after rewritten" is mean that after `Expr.substitueList ` as following. This commit is used to analyze what happen with this line ```resultExprs = Expr.substituteList(resultExprs, combinedSmap, analyzer, false);```